src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -312,12 +318,104 @@ private static void printLocales() {
             // print columns of 8
             if ((i + 1) % 8 == 0) {
                 ostream.println();
-                ostream.print(INDENT + INDENT);
+                ostream.print(TWOINDENT);
+            }
+        }
+        ostream.print("\n");
+    }
+
+    private static void printSecuritySettings(String arg) {
+        if (arg.toLowerCase(Locale.ROOT).equals("properties")) {
```

Comment:
How about a string switch with "all" and default: being the catchall. It would be easier to add others and avoid multiple calls to `toLowerCase`.
It might be a good idea for unrecognized settings to print a 1-liner reminder of the valid values, including `all`